package.json

```diff
@@ -4,7 +4,7 @@
     "start:dev": "snowpack dev",
     "start:firestore": "firebase emulators:start --import=./dev-data --only firestore",
     "build": "snowpack build",
-    "test": "jest --watch",
+    "test": "jest --watch --FIRESTORE_EMULATOR_HOST=8080",
```

Comment:
Adding `--FIRESTORE_EMULATOR_HOST=8080` won't do anything... The firebase config already gets switched to the emulator (that's how `start` works without this flag)

Reply:
there was a warning that i was trying to bypass or work around it. i also forgot to remove that.